README.md

```diff
@@ -145,7 +171,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+Question2.5.cpp
```

Comment:
# 0

I spent 20 minutes trying to get this to compile, but with no success. If you can show me it compiling in some kinbd of standard C++ compiler, I'll take another look. But (for example) I don't believe any C++ compiler allows functions to be declared inside other functions.